File: Programming-Basics-with-Python/8.For Loop - Exercise/06_oscars.py
name_actor = input()
total_score = float(input())
commission_cnt = int(input())

for n in range(0, commission_cnt):
    name_of_jury = input()
    score_from_jury = float(input())
    length_of_name = len(name_of_jury)

    current_score = (length_of_name * score_from_jury) / 2

    total_score += current_score

    if total_score >= 1250.5:
        break

if total_score >= 1250.5:
    print(f'Congratulations, {name_actor} got a nominee for leading role with {total_score:.1f}!')
else:
    needed_score = 1250.5 - total_score
    print(f"Sorry, {name_actor} you need {needed_score:.1f} more!")

File: Programming-Basics-with-JavaScript/4.For Loop/06oscars.js
function oscars(input) {
    let name = input[0];
    let points = Number(input[1]);
    let juryCount = Number(input[2]);
    let winningPoints = 1250.5;
    
    for(let i = 3; i <= (juryCount + 1) * 2; i += 2) {
      points += (input[i].length * Number(input[i + 1]) / 2);
      
      if(points >= winningPoints) {
        let result = `Congratulations, ${name} got a nominee for leading role with ${points.toFixed(1)}!`;
        console.log(result);
        break;
      }
    }
    
    if(points < winningPoints) {
      let result = `Sorry, ${name} you need ${(winningPoints - points).toFixed(1)} more!`;
      console.log(result);
    }
  }
  oscars(["Zahari Baharov",
  "205",
  "4",
  "Johnny Depp",
  "45",
  "Will Smith",
  "29",
  "Jet Lee",
  "10",
  "Matthew Mcconaughey",
  "39"])
